support-payment-api/src/main/scala/model/Currency.scala

```diff
@@ -29,6 +29,6 @@ object Currency extends Enum[Currency] with CirceEnum[Currency] {
       case USD => 10000
       case _ => 2000
     }
-    amount > maxAmount
+    amount > maxAmount * 1.04
```

Comment:
Would it be worth adding a comment here to explain this maths for future readers? Or perhaps give the 0.04 a name?

```suggestion
    val transactionCostPercentage = 0.04
    amount > maxAmount * (1 + transactionCostPercentage)
```

Reply:
maybe we shoudl use 4% constant somewhere (config) with a comment?